src/components/chips/chips.spec.js

```diff
@@ -922,7 +922,7 @@ describe('<md-chips>', function() {
           input.val('    ');
           input.triggerHandler('input');
 
-          expect(input.controller('ngModel').$modelValue).toBe('');
+          expect(input.controller('ngModel').$modelValue).toBeFalsy();
```

Comment:
IE sets `input` to `undefined` now while Chrome and FireFox still evaluate to `''`